lib/src/components/NxSmallThreatCounter/NxSmallThreatCounter.scss

```diff
@@ -67,9 +68,18 @@
       text-align: center;
       width: 100%;
 
-      &--zero {
-        visibility: visible;
+      @each $category in critical, severe, moderate, low, none, unspecified {
```

Comment:
This is repeated code from higher up right? There's probably ways to avoid that.  A `:not` selector if nothing else

Reply:
> This is repeated code from higher up right? There's probably ways to avoid that. A `:not` selector if nothing else

In [32c60dc](https://github.com/sonatype/sonatype-react-shared-components/pull/740/commits/32c60dc098f038cdc458457581334b28447078c8) I tried using @if and @ else within a @mixin. However I'm not sure if it should be inside the scss-shared folder since the conditional will just apply to this situation (and I couldn't get an @if and @else statement to work outside of a @mixin properly).